net/ddns-go/Makefile

```
@@ -0,0 +1,94 @@
+# SPDX-License-Identifier: GPL-3.0-only
+#
+# Copyright (C) 2021-2023  sirpdboy  <herboy2008@gmail.com> https://github.com/sirpdboy/luci-app-ddns-go 
+# 
+# This is free software, licensed under the Apache License, Version 2.0 .
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ddns-go
+PKG_VERSION:=5.3.6
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/jeessy2/ddns-go.git
+PKG_MIRROR_HASH:=7b865a8b9618147514c4bb6220f02d2404ab5f6586e27797b717b4d7b61b81b6
+PKG_SOURCE_VERSION:=4d33865890bffebeebb1fe49f079d67c1d8c4625
+
+PKG_LICENSE:=AGPL-3.0-only
```

```suggestion
PKG_LICENSE:=MIT
```

See [LICENSE](https://github.com/jeessy2/ddns-go/blob/v5.3.6/LICENSE).